### DISCUSSION TITLE
cv2 imwrite error

### DISCUSSION THREAD

**Discussion by Naman065Malik** [datasciencecompleteguide/Machine-Learning#9] [Q&A]
I am trying to make a face lock system for my AI (jarvis). Right now I am collecting samples of my face but there is a error. i tried to find solution on youtube but i cant see any tutorial about solution of it. please help me.
![Screenshot 2021-06-23 233503](https://user-images.githubusercontent.com/86320160/123147094-4266d200-d47c-11eb-9ae8-1a3f36e5dcc9.png)
![Screenshot 2021-06-23 233606](https://user-images.githubusercontent.com/86320160/123147130-4c88d080-d47c-11eb-9479-9981753d78f4.png)

**Comment by datasciencecompleteguide** [comment]
> 
> 
> I am trying to make a face lock system for my AI (jarvis). Right now I am collecting samples of my face but there is a error. i tried to find solution on youtube but i cant see any tutorial about solution of it. please help me.
> ![Screenshot 2021-06-23 233503](https://user-images.githubusercontent.com/86320160/123147094-4266d200-d47c-11eb-9ae8-1a3f36e5dcc9.png)
> ![Screenshot 2021-06-23 233606](https://user-images.githubusercontent.com/86320160/123147130-4c88d080-d47c-11eb-9479-9981753d78f4.png)


check the line 19 for you program
you have written:
converted_image [y : y+h,  **x : +w]**
instead you should write:
converted_image [y : y+h,  **x : x+w**]
